components/tooltip/tooltip.js

```diff
@@ -650,6 +657,27 @@ class Tooltip extends RtlMixin(LitElement) {
 		return spaceAround;
 	}
 
+	async _determineIfTruncating() {
```

Comment:
Nit: I might be inclined to name this method `_updateTruncating` to better reflect what it's doing.